src/hockeypuck/hkp/handler.go

```diff
@@ -167,6 +171,21 @@ func KeyWriterOptions(opts []openpgp.KeyWriterOption) HandlerOption {
 	}
 }
 
+func AdminKeys(adminKeys []string) HandlerOption {
+	// Normalise adminKeys to lowercase without 0x prefix on startup
+	return func(h *Handler) error {
+		for index, fp := range adminKeys {
+			if strings.HasPrefix(fp, "0x") {
```

Comment:
Consider https://pkg.go.dev/strings#TrimPrefix here

Reply:
Nice, will definitely use this.

Reply:
fixed in a7c90d2